### DISCUSSION TITLE
Is there a tutorial to install solocraft and lfg solo?

### DISCUSSION THREAD

**Discussion by Orystal** [TrinityCore/TrinityCore#27818] [Q&A]
Hi all

I'm new to Trinity core and I was wondering if there is a tutorial to install Trinity with Solocraft and LFG SOLO because every time I try to compile the two sets I get errors.
Is there also a way to import accounts / personal from Azerothcore?
Thank you

**Comment by Aokromes** [comment]
we cannot read minds.

**Reply by Faq** [reply]
@Orystal if you encountered compile errors, then post them.

**Comment by nick25252525** [comment]
I would also be interested in the Solocraft. can you paste it?

**Comment by Slotease** [comment]
The way i did it was to download the sources from the repo and unpack them into the module folder before i used cmake and compile. You are supposed to clone them but slapping the unzipped into the folder worked for me. Then rename them by removing the master at the end. After that, use cmake and then compile. On one server i had to enable them in the top cmake page if i rem correctly.  

https://www.azerothcore.org/wiki/installing-a-module
